pkg/controller/installation/core_controller.go

```diff
@@ -196,6 +198,14 @@ func add(mgr manager.Manager, r *ReconcileInstallation) error {
 		return fmt.Errorf("tigera-installation-controller failed to watch primary resource: %w", err)
 	}
 
+	// Watch for changes to TigeraStatus.
+	err = c.Watch(&source.Kind{Type: &operator.TigeraStatus{ObjectMeta: metav1.ObjectMeta{Name: "calico"}}}, &handler.EnqueueRequestForObject{}, predicate.NewPredicateFuncs(func(object client.Object) bool {
```

Comment:
For the subsequent controllers it would be nice to extract a func for this where you pass in the name. For this PR I'm ok keeping it like this.

Reply:
Sure.. Will incorporate as part of the next PR while implementing the changes for all CRs